### PR TITLE
feat: allow to uninstall several chart releases at once (#653)

### LINKED ISSUE #620: Cannot uninstall several helm charts at once
**Steps:**
1. ASSERT: have several helm charts installed
2. ASSERT: have them listed in the resource tree
3. EXEC: select 2+ helm charts in the resource tree
4. EXEC: open context menu and look for "Uninstall"

**Steps:**
There's no "Uninstall" where there are several helm charts installed. You can only uninstall if there's 1 helm chart selected.

### PR DESCRIPTION
**Pull request by adietish**
fixes #620

### CODE REVIEW

File: src/main/java/org/jboss/tools/intellij/openshift/actions/helm/UninstallReleaseAction.java [hunk 1]
```diff
@@ -17,46 +17,89 @@
...
+    return Arrays.stream(selected)
+      .map(item -> {
+        Object node = getElement(item);
+        if (node instanceof ChartReleaseNode) {
+          return ((ChartReleaseNode) node);
+        } else {
+          return null;
+        }
+      })
+      .filter(Objects::nonNull)
```
Comment (sbouchet) [inline src/main/java/org/jboss/tools/intellij/openshift/actions/helm/UninstallReleaseAction.java:91]:
```suggestion
      .map(StructureTreeAction::getElement)
      .filter(ChartReleaseNode.class::isInstance)
      .map(ChartReleaseNode.class::cast)
```
should do same

**Review by sbouchet — COMMENTED** [review]
multiselection for unistall works. wonder if we can implement that in delete components/services too ?

**Comment by adietish** [comment]
> multiselection for unistall works. wonder if we can implement that in delete components/services too ?

can you please file an issue so that we can plan this?

**Comment by sbouchet** [comment]
> > multiselection for unistall works. wonder if we can implement that in delete components/services too ?
> 
> can you please file an issue so that we can plan this?

opened #669

**Review by sbouchet — APPROVED** [review]
LGTM.

**Pull request merged**